External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -2832,6 +2832,8 @@ private static void GenerateFunctionJob(GeneratorJobType job, ScintillaControl s
             }
             if (caller?.Parameters != null && caller.Parameters.Count > 0)
             {
+                Func<string, string> cleanType = null;
+                cleanType = s => s.StartsWith("(") && s.EndsWith(')') ? cleanType(s.Trim('(', ')')) : s;

```

Comment:
But it can happen. Sadly I don't have the exact test case here, but I saw it yesterday while playing around. Anyway, merging this for now, I don't think it's that important.